### PR TITLE
Add data preservation with JavaScript pair programming project

### PR DESCRIPTION
**Pull request by juaryimami**
- In this project:
- we add a local storage feature for the contact form.
- we practiced pair programming well.
  Thank you for reviewing our project.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -113,4 +118,36 @@ form.addEventListener('submit', (e) => {
...
+    message = null;
+  } else {
+    name = JSON.parse(localSotrage.getItem('formData')).name;
+    email = JSON.parse(localSotrage.getItem('formData')).email;
+    message = JSON.parse(localSotrage.getItem('formData')).message;
+  }
+  CONTACT_NAME.value = name;
+  CONTACT_EMAIL.value = email;
+  CONTACT_MESSAGE.value = message;
+};
```
Comment (roshan-bajgain) [inline src/main.js]: - [x] When the user loads the page if there is any data in the local storage the input fields are pre-filled with this data. Please Implement it.
`

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @juaryimami, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/main.js (hunk 1 above)
Comment (juaryimami) [reply src/main.js]: thank you I will fix it

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @juaryimami 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

No linter errors. ✔️
GitHub flow is followed. ✔️
Professional README file. ✔️
PR has a good title and summary. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by juaryimami** [comment]
Thank you

**Pull request merged**